packages/generator/src/sdk-metadata/__snapshots__/sdk-metadata.spec.ts.snap

```diff
@@ -39,6 +39,6 @@ exports[`sdk-metadata generates the header content 1`] = `
 Object {
   "name": "API_TEST_SRV",
   "type": "odata",
-  "version": "1.39.0",
+  "version": "1.0.0",
```

Comment:
This snapshot relies on the current version, which will be changed after a release.